src/Employer/Employer.Web/Views/Shared/_Part1Header.cshtml

```diff
@@ -0,0 +1,74 @@
+﻿@{
+    var exampleClosingDate = DateTime.Now.AddMonths(2);

```

Comment:
We could put these in a static `ExampleVacancyViewModel`

Reply:
I think its simple enough at the moment to leave.  Good idea though if we start getting clever with the example advert.